config/config.prod.json

```diff
@@ -1,6 +1,6 @@
 {
-  "MASQ_APP_BASE_URL": "https://qwantresearch.github.io/masq-app/",
-  "HUB_URLS": [ "wss://signalhub-jvunerwwrg.now.sh" ],
+  "MASQ_APP_BASE_URL": "should_be_overridden",
```

Comment:
why ?

Reply:
Because we should provide the production url once we have it and override until then

Reply:
ah yes it's true we don't have a production url yet ;)